index.js

```diff
@@ -144,6 +157,38 @@ bot.onText(/^\/autodelete(@sticker_time_bot)?(\s+([^\s]+))?$/, (msg, match) => {
     }
 });
 
+bot.onText(/\/sleeptime (\d+)/, (msg, match) => {
+    const chatId = msg.chat.id;
+    // bot.sendMessage(chatId, match[0]+'  '+match[1]+'  '+match[2]+'  '+match[3])
+    if (match[1]) {
+        if (match[1] <= 60 && match[1] >= 0){
```

Comment:
Hour should never be larger than 23.
```suggestion
        if (match[1] <= 23 && match[1] >= 0){
```

Reply:
Yeah, used minute for demo and forget to change back